src/ui/components/filters/filterboxcomponent.js

```diff
@@ -234,18 +240,35 @@ export default class FilterBoxComponent extends Component {
   }
 
   resetFilters () {
+    const validFilterNodes = this._filterNodes.filter(fn => fn.getFilter().getFilterKey());
```

Comment:
Do we do this calculation elsewhere in the component? If so, I think we should make a getValidFilterNodes method.

Reply:
yup done in two places, added the helper method